app/views.py

```diff
@@ -0,0 +1,30 @@
+from app import app
+from flask import render_template
+
+
+@app.route('/')
+def index():
+    return render_template('index.html')
+
+
+@app.route('/<input>')
+def print_user_input(input):
```

Comment:
please don't use the `input` name. it shadows a builtin. 

Please have  alook at the list of builtins, e.g. https://docs.python.org/2/library/functions.html, to make sure you're not shadowing their names.